client/layouts/default.vue

```diff
@@ -97,6 +97,9 @@
         :fab="!(opaqueAppBar || $vuetify.breakpoint.xsOnly) && !isScrolled"
         bottom
       />
+      <cast-button
```

Comment:
The buttom needs some margin when not in fab:

![image](https://user-images.githubusercontent.com/10274099/118170059-f7dc4980-b429-11eb-8560-363d3ee68571.png)

![image](https://user-images.githubusercontent.com/10274099/118170148-0e82a080-b42a-11eb-9260-c02e24df326a.png)